lib/operations/numeric/greaterEqual.js

```diff
@@ -0,0 +1,6 @@
+function greaterEqual(a, b) {
+  return a >= b;
+}
+
+greaterEqual.code = '>=';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`>=` is an existing operator